src/OSPSuite.Infrastructure.Import/Services/Importer.cs

```diff
@@ -80,7 +80,7 @@ public void AddFromFile(IDataFormat format, Cache<string, DataSheet> dataSheets,
                data.AddData(format.Parse(sheetKeyValue.Value.RawData, columnInfos));
                dataSets.Add(sheetKeyValue.Key, data);
             }
-            catch
+            catch (Exception e)
```

Comment:
what about e? Don't need it? Then remove

Reply:
true, removed it